src/shared/XLinkDispatcherImpl.c

```diff
@@ -487,9 +487,12 @@ void dispatcherCloseLink(void* fd, int fullClose)
         while (getPacketFromStream(stream) || stream->blockedPackets) {
             releasePacketFromStream(stream, NULL);
         }
+        memset(stream, 0, sizeof(*stream));
```

Comment:
I don't see the value on this change. `XLinkStreamReset()` is only called once...here.
Yes, can move all the code from XLinkStreamReset() into this location. I do not see any value in doing it.
I see no harm, no plus, or minus.